c10/core/impl/LocalDispatchKeySet.h

```diff
@@ -37,18 +37,18 @@ struct C10_API PODLocalDispatchKeySet {
   // See Note [TLS Initialization]
   DispatchKeySet included() const {
     return DispatchKeySet(DispatchKeySet::RAW, included_) ^
-        c10::default_included_set;
+        get_default_included_set();
   }
   DispatchKeySet excluded() const {
     return DispatchKeySet(DispatchKeySet::RAW, excluded_) ^
-        c10::default_excluded_set;
+        get_default_excluded_set();
   }
 
   void set_included(DispatchKeySet x) {
-    included_ = (x ^ c10::default_included_set).raw_repr();
+    included_ = (x ^ get_default_included_set()).raw_repr();
   }
   void set_excluded(DispatchKeySet x) {
-    excluded_ = (x ^ c10::default_excluded_set).raw_repr();
+    excluded_ = (x ^ get_default_excluded_set()).raw_repr();
```

Comment:
This is now a function call; I want this to be only a memory dereference with this change. Also, you don't need a global exclude, so I wouldn't pay the cost for the memory deref for exclude unless you need it